src/EditorFeatures/Core/Structure/OutliningOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Structure
+{
+    internal static class OutliningOptions
+    {
+        public static readonly PerLanguageOption2<bool> Outlining = new("dotnet_enter_outlining_mode_when_files_open", defaultValue: true);
```

Comment:
>dotnet_enter_outlining_mode_when_files_open [](http://example.com/codeflow?start=73&length=43)

maybe also `_on_` instead of `_when_` for consistency with other option names.